www/about/policies/terms-of-service.md

```diff
@@ -89,17 +89,17 @@ Collaborator if either party breaches the terms of any such agreement (e.g. if
 the Collaborator fails to provide the promised goods or services).
 
   1. Gratipay does not determine how payments are allocated and is not
-     responsible for how Projects internally distribute payments. The agreement
-between the Project's Owner and Collaborators regarding the allocation of
+     responsible for how Project Owners distribute payments. The agreement
+between the Project's Owner and other Collaborators regarding the allocation of
 payments is solely between those parties.
 
 1. Terms for Givers
 
-  1. When you make a payment to a Project Owner, it is a payment to fund the
-     Work described in the Project's profile. You understand and acknowledge
-that your payment does not entitle you personally to receive any goods or
-services from the Project Owner, unless the Project Owner explicitly offers
-those things to Givers.
+  1. When you make a payment via a Project to a Project Owner, it is a payment
+     to fund the Work described in the Project's profile. You understand and
```

Comment:
Done in https://github.com/gratipay/gratipay.com/pull/4117/commits/9bf78ca4dde61b630495e5f34f1e018f59bb4f03.